app/models/link_checker_api_report/link.rb

```diff
@@ -0,0 +1,16 @@
+class LinkCheckerApiReport::Link < ActiveRecord::Base
+  serialize :check_errors, Hash
+  serialize :check_warnings, Hash
+
+  belongs_to :report, class_name: LinkCheckerApiReport
+
+  def self.attributes_from_link_report(payload)
```

Comment:
This uses the output of the webhook as well so it's designed to be compatible with both a strong params object and the output from GDS API - not sure if that might bite at some point though.